README.md

```diff
@@ -6,18 +6,15 @@
 [![Build and Release](https://github.com/metaplex-foundation/sugar/actions/workflows/build.yml/badge.svg)](https://github.com/metaplex-foundation/sugar/actions/workflows/build.yml)
 [![License](https://img.shields.io/crates/l/sugar-cli)](https://github.com/metaplex-foundation/sugar/blob/main/LICENSE)
 
-
 # Sugar: A Candy Machine CLI
 
 <p align="center">
   <img src="animation.gif">
 </p>
 
-
-
 Sugar is an alternative to the current Metaplex Candy Machine CLI. It has been written from the ground up and includes several improvements:
 
-- better performance for upload of media/metadata files and deploy of the candy machine &mdash; these operations take advantage of multithreaded systems to significantly speed up the computational time needed;
+- better performance for upload of image/metadata files and deploy of the candy machine &mdash; these operations take advantage of multithreaded systems to significantly speed up the computational time needed;
```

Comment:
Wonder if we should keep `media` since it represent both images and videos (animation).